pkg/api/correlation.go

```diff
@@ -1,12 +1,22 @@
 package api
 
 import (
+	"context"
+	"net/http"
 	"time"
+
+	"github.com/Azure/ARO-RP/pkg/util/uuid"
 )
 
 // Copyright (c) Microsoft Corporation.
 // Licensed under the Apache License 2.0.
 
+type contextKey int
+
+const (
+	contextKeyCorrelationData contextKey = iota
+)
+
 // CorrelationData represents any data, used for metrics or tracing.
 // More on these values:
 // https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md
```

Comment:
True. I have replaced it with: https://eng.ms/docs/products/arm/rpaas/contract/requestheaders

You can check the Spike doc for a explanation about the correlation data 🙂 

Thanks!

Reply:
So that link is only accessible to users with a B-account (AFAIK). If that's true I'm not exactly sure what my point is but it's another thing that makes having this repo public a bit... strange. :man_shrugging:

Reply:
AFAIK everything under eng.ms requires b- account.